### DISCUSSION TITLE
[@types/node] The `path/posix` and `path/win32` modules are not accessible with `node:` prefix

### DISCUSSION THREAD

**Discussion by esdmr** [DefinitelyTyped/DefinitelyTyped#57816] [Issues with a @types package] [answered]
Related file: [path.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/path.d.ts)

At the end of the file, the module `path` is re-exported as `node:path`, but the two submodules `path/posix` and `path/win32` defined at the top of the file are not re-exported.

Currently, this is the only module in `@types/node` which is not fully re-exported with the `node:` prefix.

[a comment by an automated account is not shown]

**Comment by SimonSchick** [comment, marked as answer]
Big oof, seems like an ez fix if you want to do it yourself, otherwise I will work on it some time today (maybe).

**Reply by esdmr** [reply]
Sure. I will send a PR.

Edit: Pull request at #57822.
